agent/e2e/agent-tui/utils.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# source common.sh
+set +x
+
+### utils function for sending various keys and/or text to the console
```

Comment:
For better tracking, this PR should be merged after https://github.com/openshift-metal3/dev-scripts/pull/1744 which introduces code refactoring.